rust/cubesql/cubesql/src/lib.rs

```diff
@@ -12,6 +12,7 @@
 // #![feature(trace_macros)]
 #![recursion_limit = "2048"]
 #![feature(error_generic_member_access)]
+#![feature(map_try_insert)]
```

Comment:
Will be cool to avoid new feature flags, because it requires nightly version. I am still thinking that in the next few releases with can move to rust stable.

Reply:
The feature flag has been removed.